resources/sushy-tools/redfish-emulator.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+set -eux -o pipefail
+
+CONFIG="${SUSHY_TOOLS_CONFIG:-/root/sushy/conf.py}"
+ARGS=
+
+if [[ -f "${CONFIG}" ]]; then
+    ARGS="${ARGS} --config ${CONFIG}"
+fi
+
+if [[ ! -f "${CONFIG}" ]] || ! grep -q "^SUSHY_EMULATOR_LISTEN_IP =" "${CONFIG}"; then
```

Comment:
```suggestion
elif ! grep -q "^SUSHY_EMULATOR_LISTEN_IP =" "${CONFIG}"; then
```
Avoids repetive check about config existing.

Reply:
I think you're wrong, there an "or", not an "and" here.